build.sh

```diff
@@ -52,6 +52,9 @@ dockerbuild()
     BUILD_COMMAND=/opt/code/build_projects/dotnet-cli-build/build.sh $DIR/scripts/dockerrun.sh --non-interactive "$@"
 }
 
+# Initialize the tooling
+$DIR/init-tools.sh "${args[@]}"
```

Comment:
Does this need to be run inside the docker container?  That way we get the correct stage0 for the actual platform we will be building on.